### ISSUE TITLE
Cannot reproduce the results

### ISSUE THREAD

**Issue by PatrickFeng** [JiehongLin/Self-DPDN#4]
Hi Lin,

Thanks for sharing the work.
When I trained the model on my own machine, the evaluation results are lower than the papers. I trained the model on unsupervised mode and tested it with and without mask label respectively. Below are w/ mask and w/o mask label testing results.

Test with mask label:
'''
2022-10-12 15:15:54,587 - 3D IoU at 25: 69.8
2022-10-12 15:15:54,587 - 3D IoU at 50: 41.5
2022-10-12 15:15:54,587 - 3D IoU at 75: 14.6
2022-10-12 15:15:54,587 - 5 degree, 2cm: 20.6
2022-10-12 15:15:54,587 - 5 degree, 5cm: 23.3
2022-10-12 15:15:54,587 - 10 degree, 2cm: 36.6
2022-10-12 15:15:54,587 - 10 degree, 5cm: 41.2
'''

Test without mask label:
'''
2022-10-12 13:57:20,588 - 3D IoU at 25: 60.1
2022-10-12 13:57:20,609 - 3D IoU at 50: 39.5
2022-10-12 13:57:20,609 - 3D IoU at 75: 15.0
2022-10-12 13:57:20,609 - 5 degree, 2cm: 21.3
2022-10-12 13:57:20,609 - 5 degree, 5cm: 25.5
2022-10-12 13:57:20,610 - 10 degree, 2cm: 36.5
2022-10-12 13:57:20,610 - 10 degree, 5cm: 42.0
'''

The results are not as good as that in the paper. Much appreciated if you could give me some advice.

**Comment by JiehongLin** [comment]
Thanks for your attention on our work. Before publishing the paper, I have re-trained the models by the released code and obtain the results in the paper. You could download and evaluate our trained models provided in the [Link](https://drive.google.com/file/d/1hWkbH4Z0RXQeYLxC_kOINxdTnDLtm9SX/view?usp=sharing). So I wonder if you change any settings or use the same data as us?

**Comment by PatrickFeng** [comment]
> Thanks for your attention on our work. Before publishing the paper, I have re-trained the models by the released code and obtain the results in the paper. You could download and evaluate our trained models provided in the [Link](https://drive.google.com/file/d/1hWkbH4Z0RXQeYLxC_kOINxdTnDLtm9SX/view?usp=sharing). So I wonder if you change any settings or use the same data as us?

Thanks, Lin.
I trained the model on pytorch 1.10 / cuda 11.1 and got the bad results.
When I retrained the model on pytorch 1.4.0 / cuda 10.2, I got following:

'''
2022-10-18 15:37:27,065 - 3D IoU at 25: 74.2
2022-10-18 15:37:27,065 - 3D IoU at 50: 72.1
2022-10-18 15:37:27,065 - 3D IoU at 75: 61.3
2022-10-18 15:37:27,066 - 5 degree, 2cm: 32.1
2022-10-18 15:37:27,066 - 5 degree, 5cm: 40.4
2022-10-18 15:37:27,066 - 10 degree, 2cm: 51.5
2022-10-18 15:37:27,066 - 10 degree, 5cm: 67.6
'''

It is much closer to the best results. I guess the version of pytorch matters.
